config/software/chef.rb

```diff
@@ -47,50 +47,32 @@
     # Therefore we copy the tools to the necessary places.
     # We need tar for 'knife cookbook site install' to function correctly
     {
-      'tar.exe'          => 'bsdtar.exe',
-      'libarchive-2.dll' => 'libarchive-2.dll',
-      'libexpat-1.dll'   => 'libexpat-1.dll',
-      'liblzma-1.dll'    => 'liblzma-1.dll',
-      'libbz2-2.dll'     => 'libbz2-2.dll',
-      'libz-1.dll'       => 'libz-1.dll',
+        'tar.exe'          => 'bsdtar.exe',
+        'libarchive-2.dll' => 'libarchive-2.dll',
+        'libexpat-1.dll'   => 'libexpat-1.dll',
+        'liblzma-1.dll'    => 'liblzma-1.dll',
+        'libbz2-2.dll'     => 'libbz2-2.dll',
+        'libz-1.dll'       => 'libz-1.dll',
     }.each do |target, to|
       copy "#{install_dir}/embedded/mingw/bin/#{to}", "#{install_dir}/bin/#{target}"
     end
+  end
 
-    bundle "install --without server docgen", env: env
-
-    # Install components that live inside Chef's git repo. For now this is just
-    # 'chef-config'
-    bundle "exec rake install_components", env: env
-
-    gem "build chef-{windows,x86-mingw32}.gemspec", env: env
-
-    gem "install chef*mingw32.gem" \
-        " --no-ri --no-rdoc" \
-        " --verbose", env: env
-
-    block "Build Event Log Dll" do
-      Dir.chdir software.project_dir do
-        rake = windows_safe_path("#{install_dir}/embedded/bin/rake")
-        `#{rake} -rdevkit build_eventlog"` if File.exist? "#{software.project_dir}/ext/win32-eventlog"
```

Comment:
what happened to this? Is it not needed any more or does rubygems build it for us?